packages/regl-worldview/src/stories/assertionStories/GLText.stories.js

```diff
@@ -0,0 +1,145 @@
+// @flow
+
+//  Copyright (c) 2018-present, GM Cruise LLC
+//
+//  This source code is licensed under the Apache License, Version 2.0,
+//  found in the LICENSE file in the root directory of this source tree.
+//  You may not use this file except in compliance with the License.
+
+/* eslint-disable react/display-name */
+
+import { storiesOf } from "@storybook/react";
+import expect from "expect";
+import React from "react";
+import { withScreenshot } from "storybook-chrome-screenshot";
+
+import Cubes from "../../commands/Cubes";
+import GLText from "../../commands/GLText";
+import { clickAtOrigin, WorldviewWrapper } from "../worldviewAssertionUtils";
+import { assertionTest } from "stories/assertionTestUtils";
+
+function textMarkers({
+  text,
+  billboard,
+  background = false,
+}: {
+  text: string,
+  billboard?: ?boolean,
+  background?: ?boolean,
+}) {
+  const color = { r: 1, g: 1, b: 1, a: 1 };
+  return [
+    {
+      text,
+      pose: {
+        orientation: { x: 0, y: 0, z: 0, w: 1 },
+        position: { x: 0, y: 0, z: 0 },
+      },
+      scale: { x: 1, y: 1, z: 1 },
+      color,
+      colors: background ? [color, { r: 1, g: 1, b: 0, a: 1 }] : undefined,
+      billboard,
+    },
+  ];
+}
+
+function createAssertionTest(markers, scaleInvariant) {
```

Comment:
```suggestion
function createAssertionTest({ markers, scaleInvariant }: {| markers: TextMarker[], scaleInvariant: boolean |}) {
```

I found the tests below a bit confusing to read, since they sometimes have a `, true`, but it always means something different (sometimes it means `scaleInvariant`,  sometimes it means `stackedObjectsEnabled`). It would be a lot clearer if we pass in an object since you can then easily see the name of the thing that gets enabled.

Also would be good to add some more type information so Flow can help us better.

Reply:
@janpaul123 There are two functions for creating tests:
* `createAssertionTest` accepts an optional `scaleInvariant` argument. 
* `createAssertionTestWithBackgroundObjects` accepts an optional `stackedObjectsEnabled` argument and scale invariant is not used.
But I'll add your suggestion anyway since it's more clear that way

Reply:
Yeah I understood how the code works, but it took me a while to realise it.

Reply:
Right. I pushed an updated version of createAssertionTest() that uses a single object  as parameter. As a side effect of that, there's now only one function to setup the test that handles both scale invariance and stacked objects. Thanks!